### PR TITLE
added get in tensorflow frontends

### LINKED ISSUE #6062: get

### PR DESCRIPTION
**Pull request by MahmoudAshraf97**
Hello, closes #6062 
this function is just a wrapper for ivy.deserialize according to the current tensorflow implementation, the only addition to ivy.deserialize is that when no name argument is given, linear activation is returned instead of none

### CODE REVIEW

File: ivy/functional/backends/tensorflow/activations.py [hunk 1]
```diff
@@ -74,3 +74,18 @@ def deserialize(
     name: Union[str, None], /, *, custom_objects=Union[ivy.Dict, None]
 ) -> Union[ivy.Callable, None]:
     return tf.keras.activations.deserialize(name, custom_objects)
+
+
+def get(identifier: Union[str, ivy.Callable, None], /, *, custom_objects=Union[ivy.Dict, None]) -> Union[ivy.Callable, None]:
```
Comment (WilliamHirst) [inline ivy/functional/backends/tensorflow/activations.py:79]: Here you have exceeded the maximum length for a line of code.

File: ivy/functional/backends/tensorflow/activations.py [hunk 2]
```diff
@@ -74,3 +74,18 @@ def deserialize(
...
+        return tf.keras.activations.linear
+
+    if isinstance(identifier, str):
+        identifier = str(identifier)
+        return ivy.deserialize(identifier, custom_objects=custom_objects)
+
+    elif callable(identifier):
+        return identifier
+    else:
+        raise TypeError('Could not interpret activation function identifier: {}'.format(identifier))
```
Comment (WilliamHirst) [inline ivy/functional/backends/tensorflow/activations.py:91]: Same here. This I belive you can fix by simply changing to a f-string.

**Review by WilliamHirst — CHANGES_REQUESTED** [review]
Hi Mahmoud, good job on the commit! 
There are a couple of formatting issues that need fixing.

**Comment by MahmoudAshraf97** [comment]
Hi @WilliamHirst , Changes are finished please review.

File: ivy/functional/backends/tensorflow/activations.py [hunk 3]
```diff
@@ -74,3 +74,20 @@ def deserialize(
     name: Union[str, None], /, *, custom_objects=Union[ivy.Dict, None]
 ) -> Union[ivy.Callable, None]:
     return tf.keras.activations.deserialize(name, custom_objects)
+
+
+def get(identifier: Union[str, ivy.Callable, None], /, *,
+        custom_objects=Union[ivy.Dict, None]
+) -> Union[ivy.Callable, None]:
```
Comment (WilliamHirst) [inline ivy/functional/backends/tensorflow/activations.py:81]: When splitting the function arguments over several lines, all arguments should be on a separate line. See the deserialize function in the same file for example.

File: ivy/functional/backends/tensorflow/activations.py [hunk 4]
```diff
@@ -74,3 +74,20 @@ def deserialize(
...
+        return tf.keras.activations.linear
+
+    if isinstance(identifier, str):
+        identifier = str(identifier)
+        return ivy.deserialize(identifier, custom_objects=custom_objects)
+
+    elif callable(identifier):
+        return identifier
+    else:
+        raise TypeError(f"Could not interpret activation function identifier: {identifier}")
```
Comment (WilliamHirst) [inline ivy/functional/backends/tensorflow/activations.py:93]: The line is still too long. Please split the string into several lines. Again making sure that the argument (the string) is on a separate line.

**Review by WilliamHirst — COMMENTED** [review]
Hi, there are still some formatting errors that need fixing.

**Comment by MahmoudAshraf97** [comment]
I've reformatted the function using black, please review

**Comment by WilliamHirst** [comment]
Looks good!

**Pull request merged**